src/kubernetes_cluster/spec/cluster.rs

```diff
@@ -282,12 +321,12 @@ pub open spec fn busy_kubernetes_api_rejects_request<K: ResourceView, R: Reconci
 /// This action disallows the kubernetes api to be busy from this point.
 /// This is used to constraint the crash behavior for liveness proof:
 /// the controller eventually stops rejecting requests.
```

Comment:
*the requests from the controller eventually stop being rejected by transient error